models/classes/Security/DataAccess/Repository/CachedPlatformKeyChainRepository.php

```diff
@@ -51,6 +49,10 @@ public function save(KeyChain $keyChain): void
 
     public function findAll(KeyChainQuery $query): KeyChainCollection
     {
+        if ($query->getIdentifier() == null) {
+            $query = new KeyChainQuery(PlatformKeyChainRepository::OPTION_DEFAULT_KEY_ID);
```

Comment:
This cannot use the constant value, instead we should use `getOption` from the PlatformKeyChainRepository to get proper key id, cause this can change from env to env.

Reply:
done

Reply:
thanks!